cmake/ctest/drivers/atdm/ctest-s-driver.sh

```diff
@@ -18,7 +18,7 @@ source ${WORKSPACE}/Trilinos/cmake/ctest/drivers/atdm/utils/create-src-and-build
 echo
 echo "Running: ctest -V -S $WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/ctest-driver.cmake ..."
 
-ctest -V -S \
+ctest -V -S $ATDM_CONFIG_CTEST_REGEX \
```

Comment:
I think this `$ATDM_CONFIG_CTEST_REGEX` needs to be moved to a different place in the argument list.  I think the script path needs to come directly after the `-S` argument.